library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -262,6 +263,18 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Extend the keepalive timeout when *any* frame is received on the owning HTTP/2 connection.
+   *
+   * @param h2ExtendKeepaliveTimeout whether to extend the keepalive timeout.
+   *
+   * @return This builder.
+   */
+  fun h2ExtendKeepaliveTimeout(h2ExtendKeepaliveTimeout: Boolean): EngineBuilder {
```

Comment:
ah, one thing that I noticed is that other methods related to h2 ping are named with `h2CONNECTIONKeepalive` - if you want to add it that's great, if not I think that it's not a big deal as we are going to remove this method at some point anyway (most probably)

Reply:
Yeah this is already pretty verbose that I'd rather not make it even longer.